app/notifications/validators.py

```diff
@@ -85,7 +85,13 @@ def service_has_permission(notify_type, permissions):
 def check_service_has_permission(notify_type, permissions):
     if not service_has_permission(notify_type, permissions):
         raise BadRequestError(message="Service is not allowed to send {}".format(
-            get_public_notify_type_text(notify_type, plural=True)))
+            get_public_notify_type_text(notify_type, plural=True)
+        ))
+
+
+def check_if_service_can_send_files_by_email(service_contact_link):
+    if not service_contact_link:
+        raise BadRequestError(message="Go to Service Settings to turn on sending files by email")
```

Comment:
Should this message talk about setting the service contract details?

Reply:
We discussed it with Karl and settled at: `"Send files by email is not set up yet. Go to your settings page to manage send files by email"`

Then users know where to go (the tab in settings is called `Send files by email`) and they get all the guidance they need once they get there